docs/database/use-entity-framework-db-contexts.md

```diff
@@ -0,0 +1,578 @@
+---
+title: Use Entity Framework Core with .NET Aspire
+description: Learn how to optimize the performance of .NET Aspire Entity Framework integrations using their context objects.
+ms.date: 02/21/2025
+uid: database/use-entity-framework-db-contexts
+zone_pivot_groups: entity-framework-client-integration
+---
+
+# Use Entity Framework Core with .NET Aspire
+
+In a cloud-native solution, such as those .NET Aspire is built to create, microservices often need to store data in relational databases. .NET Aspire includes integrations that you can use to ease that task, some of which use the Entity Framework (EF) Object-Relational Mapping (ORM) framework to streamline the process further.
+
+Developers use ORM frameworks to work with databases using code objects instead of SQL queries. EF automatically codes database interactions by generating SQL queries based on LINQ queries. EF supports various database providers, including SQL Server, PostgreSQL, and MySQL so it's easy to interact with relational databases while following object-oriented principles.
+
+The most commonly used .NET Aspire EF client integrations are:
+
+- SQL Server Entity Framework Core integration
+- PostgreSQL Entity Framework Core integration
+- MySQL Pomelo Entity Framework Core integration
+- Oracle Entity Framework Core integration
+- Cosmos DB Entity Framework Core integration
```

Comment:
Could you convert all these bullets into links to the corresponding articles? Also, sort them alphabetically please.

Reply:
Fixed.